specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json

```diff
@@ -465,6 +465,60 @@
         "type"
       ]
     },
+    "DatasetBZip2Compression": {
```

Comment:
We do not need to add these things back if these are not used

Reply:
For customers who deserialize payload to objects, we do not need it. But for customers who use SDK to publish pipeline to ADF/Synapse, they can still use these classes as before. Removing them will bring impacts to them. Actually we have already received internal custom's report that they rely on these types.

Reply:
I agree with @ArcturusZhang , I would not suggest to add them back. While I do acknowlege the breaking change that it causes when we removed them, it was necessarly evil based on some Swagger inconsistent before. Bringing fake classes into a Swagger serves no purpose.